strutil/strutil.go

```diff
@@ -182,3 +183,18 @@ func CommaSeparatedList(str string) []string {
 	}
 	return filtered
 }
+
+// ElliptRight returns a string that is at most n runes long,
+// replacing the last rune with an ellipsis if necessary. If N is less
+// than 1 it's treated as a 1.
+func ElliptRight(str string, n int) string {
+	if n < 1 {
+		n = 1
+	}
+	if utf8.RuneCountInString(str) <= n {
+		return str
+	}
+
+	// this is expensive; look into a cheaper way maybe sometime
+	return string([]rune(str)[:n-1]) + "…"
```

Comment:
Will we have terminal issues with the fancy unicode version of "..."?

Reply:
it's a'ight. On the default linux console (on Ubuntu) that one works. If it's got a different font loaded maybe not :man_shrugging: